### PR TITLE
Alertmanager: Add missing <title> to Overview and YAML pages

### PR DESCRIPTION
**Pull request by dtaylor113**
Added 'Alerting' `<Title />` element.

### CODE REVIEW

File: frontend/public/components/monitoring/alert-manager-config.tsx [hunk 1]
```diff
@@ -380,9 +381,14 @@ const AlertManagerConfiguration: React.FC<AlertManagerConfigurationProps> = ({ o
 export const AlertManagerConfigWrapper: React.FC<AlertManagerConfigWrapperProps> = React.memo(
   ({ obj, ...props }) => {
     return (
-      <StatusBox {...obj}>
-        <AlertManagerConfiguration {...props} obj={obj.data} />
-      </StatusBox>
+      <>
+        <Helmet>
+          <title>Alertmanager Overview</title>
```
Comment (spadgett) [inline frontend/public/components/monitoring/alert-manager-config.tsx]: We should match the page heading.

```suggestion
          <title>Alerting</title>
```

File: frontend/public/components/monitoring/alert-manager-yaml-editor.tsx [hunk 2]
```diff
@@ -97,9 +98,14 @@ export const AlertManagerYAMLEditorWrapper: React.FC<
   AlertManagerYAMLEditorWrapperProps
 > = React.memo(({ obj, ...props }) => {
   return (
-    <StatusBox {...obj}>
-      <AlertManagerYAMLEditor {...props} obj={obj.data} />
-    </StatusBox>
+    <>
+      <Helmet>
+        <title>Alertmanager YAML</title>
```
Comment (spadgett) [inline frontend/public/components/monitoring/alert-manager-yaml-editor.tsx]:
```suggestion
        <title>Alerting</title>
```

Comment (dtaylor113) [reply frontend/public/components/monitoring/alert-manager-yaml-editor.tsx]: Should this also be the case for https://github.com/openshift/console/pull/3255/files?

Comment (kyoto) [reply frontend/public/components/monitoring/alert-manager-yaml-editor.tsx]: Thanks @spadgett, @dtaylor113. I think #3255 should also use the static `Alerting` to match the page heading, so I've updated that PR.

Comment (dtaylor113) [reply frontend/public/components/monitoring/alert-manager-yaml-editor.tsx]: Thanks @spadgett & @kyoto, updated this PR to 'Alerting' as well

**Pull request merged**